opentreemap/treemap/udf.py

```diff
@@ -584,17 +584,18 @@ def delete(self, *args, **kwargs):
                          .filter(model='udf:%s' % self.pk)\
                          .delete()
 
-            for group in self.instance.mobile_api_fields:
-                if self.full_name in group.get('collection_udf_keys', []):
-                    # If this is the only collection UDF with this name,
-                    # we remove the entire group, since there would be no
-                    # eligible items to go *in* the group after deletion
-                    if len([udf for udf in udf_defs(self.instance)
-                            if udf.name == self.name]) == 1:
-                        self.instance.mobile_api_fields.remove(group)
-                    # Otherwise, just remove this UDF from the group
-                    else:
-                        group['collection_udf_keys'].remove(self.full_name)
+            if 'mobile_api_fields' in self.instance.config:
```

Comment:
As we discussed, it would be nice to have a comment here explaining what this `if` is doing.